x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks.tsx

```diff
@@ -126,10 +128,14 @@ const BenchmarkSearchField = ({
 };
 
 export const Benchmarks = () => {
+  const [pageSizes, setPageSize] = useLocalStorage<number>(
```

Comment:
```suggestion
  const [pageSize, setPageSize] = useLocalStorage<number>(
```
to match the set function, you'll also be able to remove the `pageSize: pageSizes` parts